CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v1.60.0
+------------------------------
+*August, 2019*
```

Comment:
Missing `1` 😉

Reply:
Ahhh you're too quick just spotted and pushed :P